models/category.js

```diff
@@ -5,6 +5,7 @@ module.exports = (sequelize, DataTypes) => {
   }, {});
   Category.associate = function(models) {
     // associations can be defined here
+    Category.hasMany(models.Item);
```

Comment:
After the migration is run, we can now define these new relationships:

"A Category has many Items"